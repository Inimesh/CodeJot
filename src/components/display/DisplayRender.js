import { StyleSheet, Text, View } from 'react-native'
import React from 'react'

const DisplayRender = ({ text }) => {
  return (
    <View>
      <Text>{text}</Text>
    </View>
  )
}

export default DisplayRender

const styles = StyleSheet.create({})