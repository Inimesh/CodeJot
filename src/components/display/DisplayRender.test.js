import React from 'react'
import renderer from 'react-test-renderer'

import DisplayRender from './DisplayRender'

describe('<DisplayRender />', () => { 
  it('renders children correctly', () => {
    const tree = renderer.create(<DisplayRender />).toJSON();
    expect(tree).toMatchSnapshot()
  });

})