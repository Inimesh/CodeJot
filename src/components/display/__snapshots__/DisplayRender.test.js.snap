// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DisplayRender /> renders correctly 1`] = `
<View>
  <Text />
</View>
`;
